azuredeploy.json

```diff
@@ -30,6 +30,67 @@
             },
             "type": "bool"
         },
+        "searchType": {
+            "defaultValue": "azure-search",
```

Comment:
We don't want customers to be defaulted into azure-search, so this should be none-search.